templates/organization/commissionable_asset/edit.html.twig

```diff
@@ -6,7 +6,7 @@
     <h1>Edition d'un véhicule</h1>
     {{ form_start(form) }}
     {{ form_errors(form) }}
-    {{ form_row(form.type, {label: 'Type'}) }}
-    {{ form_row(form.name, {label: 'Nom'}) }}
+    {{ form_row(form.type) }}
```

Comment:
if it's the same form as the creation one, can't you use `{{ form(form) }}` in both views in order to have all fields ?